kolibri/deployment/default/settings/rtl.py

```diff
@@ -0,0 +1,31 @@
+from __future__ import absolute_import, print_function, unicode_literals
+
+import os
+import tempfile
+
+import django.conf.locale
+
+# If KOLIBRI_HOME isn't defined in the test env, it's okay to just set a
+# temp directory for testing.
+if 'KOLIBRI_HOME' not in os.environ:
+    os.environ['KOLIBRI_HOME'] = tempfile.mkdtemp()
+
+
+from .base import *  # noqa isort:skip @UnusedWildImport
+
+LANGUAGES += [  # noqa
+    ('rt-lft', 'ʜƨilǫnƎ'),
```

Comment:
getting errors here and on line 27 running in Python 2.7:

```
SyntaxError: Non-ASCII character '\xca' in file kolibri/deployment/default/settings/rtl.py on line 17, but no encoding declared; see http://python.org/dev/peps/pep-0263/ for details
```

Adding this line to the top of the file fixes the issue:

```
# coding=utf8
```